.gitpod.yml

```diff
@@ -29,13 +33,15 @@ vscode:
   extensions:
     - golang.go
 
-workspaceLocation:
-  mattermost-gitpod-config/mattermost.code-workspace
+workspaceLocation: mattermost-gitpod-config/mattermost.code-workspace
 
 tasks:
   - name: Server
     before: |
-      cd ../mattermost/server
+      npm i -g markserv
+      markserv -p 9000 &
+      gp preview "$(gp url 9000)/welcome.md"
+      cd workspace/mattermost/server
```

Comment:
The `exit` may actually cause problems on workspace resume. And it would make it less convenient to go and debug things going on with Welcome, if the terminal exited.

Maybe we should instead have it run the foreground, and serve `welcome.md` specifically. i.e. `markserv -p 9000 welcome.md` without the `&`. Then have the port `9000` automatically open in preview when the port opens.